nameko/containers.py

```diff
@@ -385,20 +384,6 @@ def spawn_managed_thread(self, fn, protected=None, identifier=None):
         gt.link(self._handle_managed_thread_exited, identifier)
         return gt
 
-    def worker_created(self, worker_ctx):
-        """ Called after a worker is created.
-
-        This is a hook for `ServiceContainer` subclasses to track worker
-        creation.
-        """
-
-    def worker_destroyed(self, worker_ctx):
```

Comment:
it turns out that `worker_teardown` is just as effective (the patched method is run before the hook)